roles/lnls-ans-role-servweb/molecule/default/molecule.yml

```diff
@@ -0,0 +1,35 @@
+---
+dependency:
+  name: galaxy
+lint:
+  name: yamllint
+provisioner:
+  name: ansible
+  lint:
+    name: ansible-lint
+    options:
+      x: ["ANSIBLE0016"]
+  inventory:
+    group_vars:
+      default_group:
+scenario:
+  name: default
+verifier:
+  name: testinfra
+  options:
+    verbose: true
+  lint:
+    name: flake8
+driver:
+  name: docker
+platforms:
+  - name: lnls-ans-role-sirius-apps-instance
```

Comment:
I always forget to rename this to the appropriate name. This should be `lnls-ans-role-servweb-instance`.

Reply:
deleted